### ISSUE TITLE
Amanda7 is not adding most internal reviewer names at the top of their reviews (all but Mobility Bon...

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#18389]
> What application are you using?

AMANDA

> Describe the problem.

Amanda7 is not adding most internal reviewer names at the top of their reviews (all but Mobility Bond)(and sometimes also missing phone number) to Master Comment Reports (MCRs). 
This has been going on for at least 7 months but I've been too busy to create a ticket for it. Recently we have been getting questions/concerns from applicants about this issue which is why I'm entering this ticket now. 
Every file I've just reviewed is missing all but Mobility Bond Reviewer Name - the rest have no reviewer names. 
I'm also seeing that sometimes A7 is also not adding phone number for PAZ Zoning and for Drainage. I found 2 MCRs, reviewed 2 days apart by the same reviewer and their phone number shows on MCR for 2024-034308 LM but no phone number on 2024-019074 LM or 2023-063572 LM (reviewer name is not on any of the 3). With PAZ Zoning, I haven't found any with their phone number on MCR or their name. 

2 of the files I checked to create this ticket, has Mobility Bond Reviewer name SHOWING on the MCR but both are missing their phone number (2024-019074 LM, 2023-145763 LM & 2023-157465 LM). This is the only internal reviewer name I've seen on the MCR, out of all the ones I've checked for this ticket. 

> Website Address

https://amanda.austintexas.gov/backoffice/core/home.zhtml#

Internet Browser: Chrome

> Requested By
Deb C.

Request ID: DTS24-116420

**Comment by TracyLinder** [comment]
Scoping scheduled for 9/19

**Comment by TracyLinder** [comment]
- Applies to License Agreements (the template may apply to all subtypes though)

Problem(s)
- Having to manually add reviewer info to the MCR. It only happens for internal reviewers. `Reviewer name and email missing`
- The process review name might be missing for the Mobility Review Bond


**Note:**

- Q: What is the logic for external comments?
- Deb is manually adding the contact info for all external reviewers.

**Comment by TracyLinder** [comment]
### Merging of #19024 





> What application are you using?

AMANDA

> Describe the problem.

MCR on one file has printed the incorrect approved date for AE pulling the previous rejection date to the approved line. 

> Is there anything else we should know?

Problem is happening in Chrome and Edge. 
Please see the attached MCR for AE review responses. I'm also attaching an MCR that has not been altered by me (altered to show what is correct vs what A7 printed). 
Thank you!

> Website Address

https://amanda.austintexas.gov/backoffice/core/home.zhtml#

Internet Browser: Chrome

> Requested By
Deb C.

[Attachment](https://api.knack.com/v1/applications/595d00ebd315cc4cb98daff4/download/asset/66e310ce054eca02cc33a0ce/2024049109lmmcrrejected9.10.24.pdf) (136.19kb)

Request ID: DTS24-117470

**Comment by TracyLinder** [comment]
### Scoping Update

- The bugs we need to fix are on the MCR folder report.
- Next step is to review the Crystal Report field logic and be sure they are mapping correctly.

**Comment by TracyLinder** [comment]
@Lpriya-LIT will add the crystal report url to this ticket for analysis.

**Comment by Lpriya-LIT** [comment]
Update - Not able to access the report query , getting network connectivity error. Working with CTM to get the query

**Comment by TracyLinder** [comment]
@Lpriya-LIT can you please reach out to Anu to get help with this?

**Comment by Lpriya-LIT** [comment]
Update - Created Service Now ticket - INC0447548

**Comment by Lpriya-LIT** [comment]
<img width="626" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN1MwQnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--41ae9a8e86759327951e86824a40118cb3e9e649/image.png" alt="image.png" />

**Comment by Lpriya-LIT** [comment]
@TracyLinder 
Update - Jennice sent the report query , Have attached it here and sent in email as well.

**Comment by Lpriya-LIT** [comment]
[LM - 73002 - Report -Query.txt](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBemU0Qnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--c3c9879938207f351fa6eaa3baf0bd1aa4907ba7/LM%20-%2073002%20-%20Report%20-Query.txt)

**Comment by Lpriya-LIT** [comment]
<img width="1286" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMkdRQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--129fa05b032e2b126e93d0d83a5a7754dedab872/image.png" alt="image.png" />

**Comment by Lpriya-LIT** [comment]
Update - Ready for UAT on Test , the MCR document displays the reviewer name and phone number for all the processes.
TEST RSN 13264751.
Please see attached screenshot for reference

**Comment by TracyLinder** [comment]
I asked Deb to test.

**Comment by TracyLinder** [comment]
### UAT Failed & AC Update
_I'm 15 days late adding this info to the card._

**UAT Feedback (screenshot below):**
1. Not all reviews have a phone number listed on the report
2. Stakeholders would like the reviewer email auto-populate 



<img width="840" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBd3ZoQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--cd187c099b06eb31f72f1c5fd76f99df5e99287c/image.png" alt="image.png" />

**Comment by TracyLinder** [comment]
@Lpriya-LIT I updated the AC to include a new requirement to include the reviewer email, however I notice the email isn't on the template yet. Is that correct? If so would you prefer I add the enhancement to a different card?

**Comment by Lpriya-LIT** [comment]
@TracyLinder Yes Tracy , Reiewer email is not in the template , we shall Create a new card for adding it.

**Comment by Lpriya-LIT** [comment]
@TracyLinder As per our last comment , we need to create a new card to add reviewer email to the template

**Comment by TracyLinder** [comment]
@Lpriya-LIT please promote this to PROD.

**Comment by Lpriya-LIT** [comment]
Update - Changes promoted to Prod, Ready for UAT

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT @TracyLinder spoke to Deb and she said it is now working but some users are still not showing the phone numbers. I found that this is because they do not have a phone number listed in their AMANDA profiles.

Deb will get me a list of the users who dont have a phone number in the MCR and i can update them in AMANDA and close the request

Waiting on List from Deb C.

**Comment by TracyLinder** [comment]
I am closing this; it's stable on PROD. Additional work should be captured in a new ticket.
